source/content/github-application.md

```diff
@@ -0,0 +1,98 @@
+---
+title: GitHub Application (Private Beta)
+description: Pantheon's GitHub Application handles moving code from individual GitHub repositories to individual Pantheon sites.
+tags: [continuous-integration, workflow, D8, D9, D10]
+contributors: [stevector]
+contenttype: [doc]
+innav: [true]
+categories: [automate, workflows]
+cms: [drupal, wordpress]
+audience: [development]
+product: [terminus]
+integration: [--]
+reviewed: "2025-04-15"
+
+---
+
+Pantheon's GitHub application directly integrates a GitHub repository with a Pantheon site.
+This allows you to use GitHub as your code repository while still using Pantheon to run your site.
+
+The GitHub application is currently in private Beta. [Request access for your Pantheon workspace here](https://docs.google.com/forms/d/e/1FAIpQLSf0vYrRbPQBxR-hT8kGJ4bEdYPtpkTtfDvPM89xD2dNZeqLqA/viewform).
+
+Once turned, on this application accommodates a pull request workflow where a Multidev environment is created for each pull request. This allows you to test the code in the pull request before merging it.
```

Comment:
```suggestion
Once configured, this application accommodates a pull request workflow where a Multidev environment is created for each pull request. This allows you to test the code in the pull request before merging it.
```

Reply:
I think I was trying to avoid the word "configure" because there's basically no configuration to be done. What about

```suggestion
Once enabled for a site, this application accommodates a pull request workflow where a Multidev environment is created for each pull request. This allows you to test the code in the pull request before merging it.
```